package.json

```diff
@@ -7,12 +7,11 @@
     "url": "https://github.com/standardnotes/web"
   },
   "scripts": {
-    "start": "webpack-dev-server --progress",
-    "watch": "webpack --mode development -w",
-    "bundle": "webpack --mode production",
+    "start": "webpack-dev-server --progress --open --config webpack.dev.js",
```

Comment:
Do you like `--open`? I find that as soon as I want to relaunch the server or want to test something out in another browser it gets in the way.

Reply:
> Do you like `--open`?

Yeah, but I don't mind opening the browser tab manually either. I'll remove it :+1: